gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Mon Jan 14 18:31:05 IST 2019
+#Thu Jan 07 20:26:35 IST 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-all.zip
-
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

Comment:
And this one tooo what is this used for, can you explain all these 3 changes that i commented on.